File: LC/Medium/1291.py
class Solution:
    def sequentialDigits(self, low: int, high: int) -> List[int]:
        n = 10
        ans = []
        d_low = len("%i" % low)
        d_high = len("%i" % high)
        sample = "123456789"  
        
        for length in range(d_low, d_high + 1):
            for start in range(n - length):
                num = int(sample[start: start + length])
                if num >= low and num <= high:
                    ans.append(num)
        
        return ans

File: LC/C++/Medium/1291.cpp
class Solution {
public:
    vector<int> sequentialDigits(int low, int high) {
        std::string sample = "123456789";
        int n = 10;
        std::vector<int> ans{};
        for (size_t length = to_string(low).size(); length < to_string(high).size() + 1; ++length) {
            for (size_t i = 0; i < n - length; ++i) {
                int num = stoi(sample.substr(i, length));
                if (num >= low && num <= high) {
                    ans.push_back(num);
                }
            }
        }

        return ans;
    }
};